addons/account/models/account_reconcile_model.py

```diff
@@ -387,6 +387,8 @@ def _get_write_off_move_lines_dict(self, st_line, residual_balance):
                 if not line.force_tax_included:
                     for tax_line in tax_vals_list:
                         residual_balance -= tax_line['balance']
+            else:
+                writeoff_line['tax_ids'] = []
```

Comment:
Could you directly add it in the `writeoff_line` dict and, maybe, make line 379 be `+=` ?

Reply:
@Levizar Done, thanks for the suggestion